app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+  def search
+    @query = params[:q]
+    @hits = FoodDataService.search(@query)[:totalHits]
+    @display_foods = FoodFacade.search(params[:q])[0..9]
+  end
```

Comment:
Nice and light controller! Would have loved to see you only hit the service once, and only create the number of Food objects that you intend to display.